### PR TITLE
Re-enable CSV end-to-end test in postgres mode

### PR DESCRIPTION
**Pull request by rhybrillou**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Review by md2119 — COMMENTED** [review]
The csv export endpoint `/api/vm/export/csv`has been deprecated on Postgres. The new endpoints are `/api/export/csv/image/cve`, `/api/export/csv/node/cve`, and `/api/export/csv/cluster/cve`

File: qa-tests-backend/src/test/groovy/CSVTest.groovy [hunk 1]
```diff
@@ -184,7 +184,11 @@ class CSVTest extends BaseSpecification {
     }
 
     def getComponentQuery() {
-        return "COMPONENT ID:" + getComponentId() + "+Fixable:true"
+        return "COMPONENT ID:" + getComponentId() + "+" + isPostgresRun() ? "Is" : "" + "Fixable:true"
+    }
+
+    def getFixableQuery() {
+        return isPostgresRun() ? "Is" : "" + "Fixable:true"
```
Comment (janisz) [inline qa-tests-backend/src/test/groovy/CSVTest.groovy:191]: Why can't we use the same query for PG and nonPG?

```
"" + "Fixable:true"
```
Looks weird

Comment (rhybrillou) [reply qa-tests-backend/src/test/groovy/CSVTest.groovy:191]: All postgres vulnerability graphql resolvers have this line : 
logErrorOnQueryContainingField(query, search.Fixable, "IsFixable")

I am under the impression that these cause this suspicious log in central:
```
graphql/resolvers: 2022/09/01 17:53:17.566701 utils.go:366: Error: Unexpected field (Fixable) found in query to resolver (IsFixable). Response maybe unexpected.
```

**Review by charmik-redhat — APPROVED** [review]

**Pull request merged**